setup.py

```diff
@@ -8,6 +8,16 @@
 README_FILE = os.path.join(PROJECT_ROOT, "README.md")
 VERSION_FILE = os.path.join(PROJECT_ROOT, "bambi", "version.py")
 REQUIREMENTS_FILE = os.path.join(PROJECT_ROOT, "requirements.txt")
+REQUIRED_PYTHON_VERSION = (3, 7, 2)
```

Comment:
Being nitpick here, but I think that `MINIMUM_PYTHON_VERSION` is better than `REQUIRED_PYTHON_VERSION`. The latter can be interpreted as the only version where Bambi works. The former means what we need it to be, the minimum version of Python where Bambi works.

Reply:
Sure, I see how this could be confusing